src/js/controller/AppController.js

```diff
@@ -129,7 +130,10 @@ L.Dropchop.AppController = L.Class.extend({
             return this._handleFormSubmit( opsConfig, e );
         } else {
             var form = this.forms.render( e.action, config );
-            form.on( 'submit', this._handleFormSubmit.bind( this, opsConfig ) );
+            if (!this.clicked) {
+                this.clicked = true;
+                form.on( 'submit', this._handleFormSubmit.bind( this, opsConfig ) );
+            }
```

Comment:
Okay, that's what I figured it would do. The whole solution felt like a hack :wink: